website/database.py

```diff
@@ -30,10 +30,45 @@
     dynamo.Index('teacher'),
     dynamo.Index('link'),
 ])
+
+# A custom teacher adventure
+# - id (str): id of the adventure
+# - content (str): adventure text
+# - creator (str): username (of a teacher account, hopefully)
+# - date (int): timestamp of last update
+# - level (int | str): level number, sometimes as an int, sometimes as a str
+# - name (str): adventure name
+# - public (bool): whether it can be shared
 ADVENTURES = dynamo.Table(storage, "adventures", "id", indexes=[dynamo.Index("creator")])
 INVITATIONS = dynamo.Table(
     storage, "class_invitations", partition_key="username", indexes=[dynamo.Index("class_id")]
 )
+
+# Class customizations
+#
+# Various columns with different meanings:
+#
+# These I'm quite sure about:
+#
+# - id (str): the identifier of the class this customization set applies to
+# - levels (int[]): the levels available in this class
+# - opening_dates ({ str -> str }): key is level nr as string, value is an ISO date
+# - other_settings (str[]): string list with values like "hide_quiz", "hide_parsons"
+# - sorted_adventures ({ str -> { from_teacher: bool, name: str }[] }):
+#     for every level (key as string) the adventures to show, in order. If from_teacher
+#     is False, the name of a built-in adventure. If from_teacher is true, name is the
+#     id of a adventure in the ADVENTURES table. The id may refer to an adventure that
+#     has been deleted. In that case, it should be ignored.
+#
+# These not so much:
```

Comment:
Hi Rico! Thank you for taking care of this bug. This fields represent the following:

1. `level_thresholds`: the grade threshold to unlock the next level. I think it could be changed to just a field. Since it now only have the quiz field, maybe it could be changed to only be an int, instead of an object.

2. `adventures`: yes this field is deprecated, and when a teacher tries to customize a class that have the old method of saving customizations, there's a script that automatically converts them to the new way of saving them https://github.com/hedyorg/hedy/blob/1a2b9a4c7662e003d1d5d1be1e0b4771ddd2ac85/website/for_teachers.py#L221